rewrite-java-test/src/test/java/org/openrewrite/java/JavaParserTest.java

```diff
@@ -154,15 +154,52 @@ void shouldResolvePathUsingPublicClasses(@Language("java") String source) {
         rewriteRun(
           java(
             source,
-            spec -> spec.afterRecipe(cu -> assertThat(cu.getSourcePath()).isEqualTo(Path.of("my","example","PublicClass.java")))
+            spec -> spec.afterRecipe(cu -> assertThat(cu.getSourcePath()).isEqualTo(Path.of("my", "example", "PublicClass.java")))
           )
         );
     }
 
     @Test
     @Issue("https://github.com/openrewrite/rewrite/issues/1895")
-    void moduleInfo(){
+    void moduleInfo() {
         // Ignored until properly handled: https://github.com/openrewrite/rewrite/issues/4054#issuecomment-2267605739
         assertFalse(JavaParser.fromJavaVersion().build().accept(Path.of("src/main/java/foo/module-info.java")));
     }
+
+    @Test
+    void shouldParseComments() {
+        rewriteRun(
+          java(
+            """
+              class A {
+                  /*
+                   * public Some getOther() { return other; }
+                   *
+                   *//**
+                   * Sets the value of the other property.
+                   *
+                   * @param value allowed object is {@link Some }
+                   *
+                   *//*
+                   * public void setOther(Some value) { this.other =
+                   * value; }
+
+                   *//*
+                   public Some createSome() {
+                       return new Some();
+                   }
+
+                  *//**
+                   * Create an instance of {@link Some }
+                   *
+                   *//*
+                   public Some createSome2() {
+                       return new Some();
+                   }
+                  */
```

Comment:
Is this the absolute minimal to reproduce the issue? Or is this multiple variant?
Asking since stepping through with the debugger might be more complicated with multiple doc blocks in sequence.